lib/src/main/java/com/squareup/cycler/Recycler.kt

```diff
@@ -381,12 +381,25 @@ class Recycler<I : Any> internal constructor(
 
     /**
      * General method which is called from Extension code. DON'T USE DIRECTLY.
+     * Adds an [ExtensionSpec] to the [Config].
      * @hide
      */
     fun extension(spec: ExtensionSpec<I>) {
       extensionSpecs.add(spec)
     }
 
+    /**
+     * General method which is called from Extension code. DON'T USE DIRECTLY.
```

Comment:
Who is the "DON'T USE DIRECTLY" speaking to? Who shouldn't use it, and who should? _Somebody_ must be intended to use it, or it wouldn't need to exist.

Reply:
Gotcha. It's for extensions code (edges, sticky headers). Not for using directly from UI code.

Reply:
Thanks for updating the comment! LG now